### ISSUE TITLE
Unable to build locally

### ISSUE THREAD

**Issue by hlfernandez** [scottyhardy/docker-wine#52]
**Describe the bug**
I am try to build the image locally following the README instructions.

**To Reproduce**
1. Go to `docker-wine`
2. Run `make`
3. The following error during the build occurs:

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 winehq-stable : Depends: wine-stable (= 4.0.1~bionic)
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c wget https://dl.winehq.org/wine-builds/winehq.key     && apt-key add winehq.key     && apt-add-repository "deb https://dl.winehq.org/wine-builds/ubuntu/ bionic main"     && dpkg --add-architecture i386     && apt-get update     && apt-get install -y --install-recommends winehq-${WINEBRANCH}="${WINE_VER}"     && rm -rf /var/lib/apt/lists/*     && rm winehq.key' returned a non-zero code: 100
Makefile:7: recipe for target 'build' failed
make: *** [build] Error 100
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.4 LTS (4.15.0-91-generic)

**Comment by scottyhardy** [comment]
Hi @hlfernandez 

This was a known issue as I hadn't updated the repo for a while, but I actually updated the master branch only last night with a build that works.  The Dockerfile specifies the version of Wine to use as the other dependencies such as mono and gecko need to also use a compatible version (and so that builds hopefully don't introduce breaking changes), but Ubuntu don't hold onto older versions of packages for very long.

TL;DR - just use the latest version in the master branch and it should now build for you

**Comment by hlfernandez** [comment]
Thank you very much, I have been able to do the build with success now.
